suite-common/icons/assets/trezorSafe5.svg

```diff
@@ -1 +1 @@
-<svg width="32" height="32" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M7 6a4 4 0 0 1 4-4h10a4 4 0 0 1 4 4v12.191a9 9 0 0 1-1.204 4.498l-3.353 5.81A3 3 0 0 1 17.845 30h-3.69a3 3 0 0 1-2.598-1.5l-3.353-5.811A9 9 0 0 1 7 18.19V6zm4-2a2 2 0 0 0-2 2v12.191a7 7 0 0 0 .937 3.498l3.352 5.81a1 1 0 0 0 .866.501h3.69a1 1 0 0 0 .866-.5l3.352-5.81A7 7 0 0 0 23 18.19V6a2 2 0 0 0-2-2H11z" fill="#151719"/><path d="M16.983 14.667a1.335 1.335 0 0 0-2.276-.943 1 1 0 0 1-1.694-.868l.65-4.016.019-.089A1 1 0 0 1 14.65 8H18a1 1 0 1 1 0 2h-2.499l-.22 1.354a3.335 3.335 0 0 1 3.14 1.46 3.335 3.335 0 0 1-1.495 4.932 3.333 3.333 0 0 1-3.633-.723 1 1 0 0 1 1.414-1.414 1.334 1.334 0 0 0 2.276-.942z" fill="#151719"/></svg>
\ No newline at end of file
+<svg width="32" height="32" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M7 6a4 4 0 0 1 4-4h10a4 4 0 0 1 4 4v12.191a9 9 0 0 1-1.204 4.498l-3.353 5.81A3 3 0 0 1 17.845 30h-3.69a3 3 0 0 1-2.598-1.5l-3.353-5.811A9 9 0 0 1 7 18.19V6zm4-2a2 2 0 0 0-2 2v12.191a7 7 0 0 0 .937 3.498l3.352 5.81a1 1 0 0 0 .866.501h3.69a1 1 0 0 0 .866-.5l3.352-5.81A7 7 0 0 0 23 18.19V6a2 2 0 0 0-2-2H11z" fill="#000"/></svg>
```

Comment:
Looking at the other icon SVGs, they don't have `width="32" height="32"` but `viewBox="0 0 32 32"`. 
Maybe that could be the cause of the issue.

Reply:
I think the reason is that I forgot to run generate icons script 😒

Reply:
@martykan could you check it again please? 🙇‍♂️